zelda-totk/zelda-totk.class.pouch.js

```diff
@@ -202,6 +202,26 @@ Pouch.updateItemIcon=function(item){
 	else
 		item._htmlIcon.src=ICON_PATH+item.category+'/'+item.id+'.png';
 };
+Pouch.getItemIcon = function (item) {
```

Comment:
This is a good id!
Although, it is repeating code from updateItemIcon, will remove code from the former.